components/cdc/src/delegate.rs

```diff
@@ -908,6 +908,16 @@ impl Delegate {
         // To inform transaction layer no more old values are required for the region.
         self.txn_extra_op.store(TxnExtraOp::Noop);
     }
+
+    // if raw data and tidb data both exist in this region, it will return false.
+    pub fn is_raw_region(&self) -> bool {
```

Comment:
Suggest to add unit test for this new method.

Reply:
@pingyu All comments are aderessed, PTAL~